package.json

```diff
@@ -57,6 +57,7 @@
     "hyperswarm": "^4.3.6",
     "random-access-memory": "^6.1.0",
     "random-access-memory-overlay": "^3.0.0",
+    "range-parser": "^1.2.1",
```

Comment:
Dev dependency, maybe I should leave it there as it's used in the examples folder and it's unexpected to run the file to result in module not found

Reply:
The idea was to inline it but there is no much gain in doing so, unless we make a static helper that simplifies handling the Response object, setting the headers in relation to the ranges, etc but that sounds like a new tiny lib rather that inlining it in Hypercore

Reply:
sure, dont inline it